fetchData.js

```diff
@@ -0,0 +1,132 @@
+const path = require('path');
```

Comment:
Should we rename this file `update-data.js` to match `npm run update-data`?   Might slightly lower our future cognitive load.

Reply:
yep, definitely.  I kept thinking about doing that too; just forgot.  `Update` is better than `Fetch` anyway since we're persisting changes, plus consistency is a winner in either case.